ocaml/sdk-gen/go/templates/Methods.mustache

```diff
@@ -8,10 +8,10 @@
 {{#errors}}
```

Comment:
I don't see a licence placeholder in this file or SessionMethod.mustache, was it forgotten?

Reply:
The license placeholder is added in FileHeader.mustache. This header is for all generated .go files except jsonrpc_client.go

Reply:
I checked all generated .go files, all contained the license in file header.